### PR TITLE
Add lookup support.

### PR DESCRIPTION
**Pull request by derekcaneja**
Add support for checking if a number is a mobile number using twilio lookups api.
Do not send texts to non-mobile numbers.

### CODE REVIEW

File: sms_twilio.js [hunk 1]
```diff
@@ -14,6 +14,38 @@ sms.configure = function (options) {
...
 };
 
+var lookups = new Meteor.Collection('meteor_accounts_sms_lookups');
+
+/**
+ * Lookup phone number information through twilio lookups api.
+ * @param phone
+ */
+sms.lookup = function (phone) {
+  var lookup = lookups.findOne({ phone_number: { $regex: phone }});
```
Comment (raix) [inline sms_twilio.js:24]: Maybe have a check prior making sure phone is a non empty string.

Not sure about the regex match - `1` would match `111` - any special use case you had in mind?

**Comment by raix** [comment]
Just added some notes, looking good @derekcaneja

File: sms_twilio.js (hunk 1 above)
Comment (derekcaneja) [inline sms_twilio.js:24]: So the regex match is to get around having to enter +1. 
555 555555 matches +1 555 5555555.

File: sms_twilio.js [hunk 2]
```diff
@@ -14,6 +14,38 @@ sms.configure = function (options) {
...
+      Type: 'carrier'
+    }
+  });
+
+  check(response.data, {
+    country_code: String,
+    phone_number: String,
+    national_format: String,
+    url: String,
+    carrier: Object
```
Comment (jperl) [inline sms_twilio.js:41]: You should check the carrier format as well.

File: sms_twilio.js (hunk 1 above)
Comment (raix) [inline sms_twilio.js:24]: ok, accounts package api requires the use of `+1` maybe we should do the same here? I'm thinking the correct nr. is with the land code. Let me know what you think

Comment (jperl) [inline sms_twilio.js:24]: We should pass around international numbers always. So no need to do the regex.

**Pull request merged**